app/javascript/dashboard/routes/dashboard/settings/inbox/components/BusinessDay.vue

```diff
@@ -0,0 +1,225 @@
+      />
+    </div>
+    <div class="day">
+      <span>{{ dayName }}</span>
+    </div>
+    <div v-if="isDayEnabled" class="hours-select-wrap">
+      <div class="hours-range">
+        <multiselect
+          v-model="fromTime"
+          :options="timeSlots"
+          deselect-label=""
+          select-label=""
+          selected-label=""
+          placeholder="Choose"
+          :allow-empty="false"
+        />
+        <div class="separator-icon">
+          <i class="ion-minus-round" />
+        </div>
+        <multiselect
+          v-model="toTime"
+          :options="timeSlots"
+          deselect-label=""
+          select-label=""
+          selected-label=""
+          placeholder="Choose"
+          :allow-empty="false"
+        />
+      </div>
+      <div v-if="hasError" class="date-error">
+        <span class="error">{{
+          $t('INBOX_MGMT.BUSINESS_HOURS.DAY.VALIDATION_ERROR')
+        }}</span>
+      </div>
+    </div>
+    <div v-else class="day-unavailable">
+      <span>
+        {{ $t('INBOX_MGMT.BUSINESS_HOURS.DAY.UNAVAILABLE') }}
+      </span>
+    </div>
+    <div>
+      <span v-if="isDayEnabled && !hasError" class="label">
+        {{ totalHours }} {{ $t('INBOX_MGMT.BUSINESS_HOURS.DAY.HOURS') }}
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+import parse from 'date-fns/parse';
+import differenceInMinutes from 'date-fns/differenceInMinutes';
+import { generateTimeSlots } from '../helpers/businessHour';
+
+const timeSlots = generateTimeSlots(30);
+
+export default {
+  components: {},
+  props: {
+    dayName: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    timeSlot: {
+      type: Object,
+      default: () => ({
+        from: '',
+        to: '',
+      }),
+    },
+  },
+  computed: {
+    timeSlots() {
+      return timeSlots;
+    },
+    isDayEnabled: {
+      get() {
+        return this.timeSlot.from && this.timeSlot.to;
+      },
+      set(value) {
+        const newSlot = value
+          ? {
+              ...this.timeSlot,
+              from: timeSlots[0],
+              to: timeSlots[16],
+            }
+          : {
+              ...this.timeSlot,
+              from: '',
+              to: '',
+            };
+        this.$emit('update', newSlot);
+      },
+    },
+    fromTime: {
+      get() {
+        return this.timeSlot.from;
+      },
+      set(value) {
+        this.$emit('update', {
+          ...this.timeSlot,
+          from: value,
+        });
+      },
+    },
+    toTime: {
+      get() {
+        return this.timeSlot.to;
+      },
+      set(value) {
+        this.$emit('update', {
+          ...this.timeSlot,
+          to: value,
+        });
+      },
+    },
+    fromDate() {
+      return parse(this.fromTime, 'hh:mm a', new Date());
+    },
+    toDate() {
+      return parse(this.toTime, 'hh:mm a', new Date());
+    },
+    totalHours() {
+      return differenceInMinutes(this.toDate, this.fromDate) / 60;
```

Comment:
Should we use toFixed here?

Reply:
This is mainly to show total hours,  to show 7.5 hours